RetrospectiveExtension.Frontend/config/config.ts

```diff
@@ -0,0 +1,22 @@
+export interface AppConfiguration {
+    CollaborationStateServiceUrl: string,
+    AppInsightsInstrumentKey: string,
+    CurrentEnvironment: string,
+}
+function getConfiguration(): AppConfiguration {
+    console.log(`backend service URL is ${process.env.REACT_APP_COLLABORATION_STATE_SERVICE_URL}`);
+    const appInsightsInstrumentKey = process.env.REACT_APP_APP_INSIGHTS_INSTRUMENTATION_KEY;
+    const collaborationStateServiceUrl = process.env.REACT_APP_COLLABORATION_STATE_SERVICE_URL;
+    if (!appInsightsInstrumentKey) {
+        console.warn(`missing Application Insights Instrumentation key, logging and telemetry will not be collected`);
+    }
+    if (!collaborationStateServiceUrl) {
+        console.warn(`Backend Service URL was not provided, the real time updates feature will be disabled`);
+    }
+    return {
+        AppInsightsInstrumentKey: appInsightsInstrumentKey || '',
+        CollaborationStateServiceUrl: collaborationStateServiceUrl || 'https://wesbite-unset.com/',
```

Comment:
Why did you choose this value: 'https://wesbite-unset.com/'? Is there a typo in the url?

Reply:
There is a problem with the application currently which is if the URL is invalid, the entire application ceases to work, There is an exception thrown somewhere and the URL is not validated. However it is ok if the URL is not reachable ( as long as it is a valid URL). Hence I put a dummy URL. I captured this bug [here](https://dev.azure.com/csebostoncrew/Retrospectives-AzDO-Extension/_workitems/edit/1595)

Reply:
Can we fix the type then? It should be `website` right?